Zhongli.Bot/Modules/LoggingModule.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Discord.Commands;
```

Comment:
I guess this got committed accidentally

Reply:
I can remove this, but it's causing a build error for me:
![image](https://user-images.githubusercontent.com/966624/125172139-bdaed000-e16c-11eb-9d42-d054e4d456ae.png)

Reply:
Excluded this as requested.